rmgpy/data/kinetics/family.py

```diff
@@ -4049,6 +4049,23 @@ def fix_labels_mol(mol, root_labels):
                 if atm.label not in root_labels:
                     atm.label = ''
 
+        def get_react_thermo(react,rxn_react,metal):
```

Comment:
From the function name and parameter names I can't guess what this is supposed to do. The meanings of `react`, `rxn_react`, and `metal` are hard to guess out of context. Maybe consider renaming them, and/or add a docstring to the function saying what it does?

Reply:
@rwest Good point! Thanks